tests/core/pyspec/eth2spec/test/altair/transition/test_activations_and_exits.py

```diff
@@ -127,10 +130,6 @@ def test_transition_with_one_fourth_exiting_validators_exit_at_fork(state,
         assert not post_spec.is_active_validator(validator, post_spec.get_current_epoch(state))
     assert not post_spec.is_in_inactivity_leak(state)
 
-    # ensure that none of the current sync committee members are exited validators
-    exited_pubkeys = [state.validators[index].pubkey for index in exited_indices]
-    assert not any(set(exited_pubkeys).intersection(list(state.current_sync_committee.pubkeys)))
```

Comment:
I don't remember why we needed it. 😬

Reply:
there was a test where we did want to ensure some had exited, and this was the complement case where we want to ensure none in the committee are exited

Reply:
unless i'm missing something, we should keep this functionality

Reply:
@ralexstokes thanks! Now I remember why we added it. Fixed it in https://github.com/ethereum/consensus-specs/pull/2706/commits/f0bb032eb17ae441429405b89b7c158623f865c0